### ISSUE TITLE
folder downloader NameError

### ISSUE THREAD

**Issue by gabboraron** [evtimovi/google_drive_download#1]
Hi, I just want to try your code to download a folder and I got a name error:

```
Traceback (most recent call last):
  File "folder_down.py", line 30, in <module>
    pageToken=list_of_files['nextPageToken']
NameError: name 'list_of_files' is not defined
```

**Comment by evtimovi** [comment]
Good point; the `list_of_files` should be initialized earlier or we should have a check if this is the first time through the loop, in which case we can just pass an empty `pageToken`. I am not actively maintaining this repo but if you send a PR, I'll accept it :)
